servers/intro_course/utils/getCoreUrl.go

```diff
@@ -0,0 +1,13 @@
+package utils
+
+import "strings"
+
+func GetCoreUrl() string {
```

Comment:
This will most likely fail in a docker env. Since you assume https only - containers can no longer communicate with each other

Reply:
I'd even argue that this logic is not necessary. I think it is enough to assume a correct value in the env.